server/app.py

```diff
@@ -2,16 +2,39 @@
 from flask_restful import Api
 from resources.foo import Foo, testMySql, testMongo
 import common.util
+import datetime
+import logging
 
 app = Flask(__name__,
     static_folder="../static/public",
     template_folder="../static"
     )
+logging.basicConfig(level=logging.DEBUG,
+					format="%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s")
 
 api = Api(app)
 api.add_resource(Foo, '/')
 api.add_resource(testMySql, '/mysql')
 api.add_resource(testMongo, '/mongo')
 
+# Invoked after every requests to log the timestamp, content & status
+@app.after_request
+def log_request(response):
+    time = datetime.datetime.now()
+    body = response.data.decode("utf-8")
+    status_as_string = response.status
+    status_as_integer = response.status_code
+    try:
+        _id = common.util.mongo_log.db.logs.insert({
```

Comment:
can change the import to 
`from common.util import mongo, mongo_log`

then this line can be 
`_id = mongo_log.db.logs.insert({`